recipe/build.sh

```diff
@@ -25,7 +25,8 @@ cmake -DCMAKE_INSTALL_PREFIX="${INSTALL_PREFIX}" \
       -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
       -DBUILD_TESTS=${BUILD_TESTS} \
       -DBUILD_BENCHMARKS=${BUILD_BENCHMARKS} \
-      -DCPM_LOCAL_PACKAGES_ONLY=1 \
```

Comment:
... because I was being dumb? 😂

I understand it better now. `CPMAddPackage` is recursive, so `-DCPM_<package>_SOURCE=...` would make `CPMAddPackage` [stop the search and call itself with the variable undefined](https://github.com/cpm-cmake/CPM.cmake/blob/3fee69c894cbf62cd406eee3b3f8c2aa721a8bea/cmake/CPM.cmake#L588-L603), then `-DCPM_LOCAL_PACKAGES_ONLY=1` would [populate all internal variables and exit the function](https://github.com/cpm-cmake/CPM.cmake/blob/3fee69c894cbf62cd406eee3b3f8c2aa721a8bea/cmake/CPM.cmake#L616-L630). Seems to be working fine now.